### ISSUE TITLE
Regression - Utawarerumono: Itsuwari no Kamen [BLJM61288] 

### ISSUE THREAD

**Issue by marcussacana** [RPCS3/rpcs3#5098]
Basically you got a black screen when a cutscene finish or freeze if you try skip it.
Tried with the v1.0 and 1.2 of the game, same problem in both version.

Tested Version: v0.0.5-7320-fb5cdf97
Log: [rpcs3.log.zip](https://github.com/RPCS3/rpcs3/files/2363030/rpcs3.log.zip)

Specs:
PNY GTX 960 (4GB)
Intel i5 4460
8GB RAM
Win 10 v1803 - Build 17134.288

**Comment by elad335** [comment]
If its a regression, find the pull request where it regressed.

**Comment by marcussacana** [comment]
Okay~

**Comment by marcussacana** [comment]
Well... this is my first time that i need do this so, i'm not sure if I did right...
After, 2 hours exploring the commit history
I found here,
![https://pomf.pyonpyon.moe/bunlfm.png](https://pomf.pyonpyon.moe/bunlfm.png)
It's commits from: #4336 and #4422
The build from this 4336 don't have any problem 
In the first commit of the 4422, the same problem of the newest build occurs
(I rebuilded to have sure of the commit [My AppVeyor](https://ci.appveyor.com/project/marcussacana/rpcs3/build/0.0.5-6665/artifacts))
And well... I tried compare the changes bettewen this 2 commit just to have sure... [See here](https://github.com/RPCS3/rpcs3/compare/3c561633c033713ef7d08532664ae414212a84a7...rpcs3:8ca33bcb94617c8caf64a7cd7a31b4ad0a5952a3)

Well.. I did right?

**Comment by elad335** [comment]
yea you did fine. SPU Interpreter works?

**Comment by marcussacana** [comment]
Well, I tried again here and I saw I commented something wrong, 
in the first commit of that pr the problem don't occurs, in the last commit of this pr same he occurs...
And yes, SPU Interpreter works.

**Comment by marcussacana** [comment]
A update, I don't know when, but the 'problem' is partially fixed
that bug I reported before occoured in the other game of the trilogy too,
But in the master now it's paritally fixed,
in the [BLJM61331] the bug don't occours in the ASMJIT and LLVM, so, it's fixed
in the [BLJM61288] the bug don't occours in the ASMJIT... but now, in the LLVM the cutscene don't start...
And in both games the cutscene audio have a 'lag' when disable the 'lower spu thread priority' (before woked fine without enable it)


To summarize everything:
Basically, in the bug is fixed, but in the LLVM have a new regression.
And the audio is laggy

It's better close this issue? or Search where LLVM/Audio problem has started and report here?
